include/ibamr/IIMethod.h

```diff
@@ -631,6 +639,7 @@ class IIMethod : public IBStrategy
     std::vector<bool> d_use_discon_elem_for_jumps = { false };
     std::vector<bool> d_use_tangential_velocity = { false };
     std::vector<bool> d_normalize_pressure_jump = { false };
+    bool d_use_current_mesh_configuration_for_interactions = true;
```

Comment:
It might ultimately make sense to make this an `enum` --- e.g. it could be the reference coordinates, initial coordinates, current coordinates, or coordinates coming from an external model --- but this is OK for now.